build.gradle

```diff
@@ -61,6 +61,8 @@ allprojects {
         google()
         mavenCentral()
         jcenter()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
```

Comment:
I don't think you need this anymore as the Compose-specific content in kotlin-stdlib has landed in 1.3.61

Reply:
Good catch, I'll remove.